pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/Application.kt

```diff
@@ -31,12 +35,23 @@ fun KoinApplication.loadKoinModules(environment: ApplicationEnvironment): KoinAp
         //  Create a CosmosDB config that can be dependency injected (for health checks)
         single(createdAtStart = true) { CosmosConfiguration(uri, authKey) }
     }
-    val asbConfigModule = module {
-        // Create an azure service bus config that can be dependency injected (for health checks)
-        single(createdAtStart = true) { AzureServiceBusConfiguration(environment.config, configurationPath = "azure.service_bus") }
-    }
 
-    return modules(listOf(cosmosModule, asbConfigModule))
+    val configModule = module {
+        when (msgType) {
+            MessageSystem.AZURE_SERVICE_BUS.toString() -> {
+                single(createdAtStart = true) {
+                    AzureServiceBusConfiguration(environment.config, configurationPath = "azure.service_bus") }
+
+            }
+            MessageSystem.RABBITMQ.toString() -> {
+                single(createdAtStart = true) {
+                    RabbitMQService(environment.config, configurationPath = "rabbitMQ")
```

Comment:
For clarity, similar to the ASB service configuration we may want to rename this to `RabbitMQServiceConfiguration`.  Also, this will align with naming for `AzureServiceBusConfiguration`.